gptables/core/gptable.py

```diff
@@ -1,4 +1,5 @@
 import pandas as pd
+import re
```

Comment:
Does "re" this need to be added to requirements / setup?

Reply:
`re` is an inbuilt module, so does not need to be added as a requirement